includes/cleanup.php

```diff
@@ -31,19 +43,43 @@ function pmpro_delete_user_form_notice( $current_user, $userids ) {
 		}
 	}
 
+	$member_history = $wpdb->get_var( "SELECT COUNT(*) as members FROM $wpdb->pmpro_memberships_users WHERE user_id IN (" . implode( "," , $userids ) . ")" );
```

Comment:
Let's use $wpdb->prepare here.

Reply:
Create a var like userids_string and pass that as the param to prepare. Because user IDs are ints (in MySQL) we shouldn't need quote around the string. Just the commas.